package.json

```diff
@@ -9,6 +9,7 @@
   },
   "scripts": {
     "build": "tsc",
+    "lint": "eslint src",
     "test-install": "npm run build && npm -g install .",
```

Comment:
Maybe it would make sense to include some CI in the Github Actions for these?

Reply:
Yes, good point. I guess I wanted to save a little time, but having proper CI is a big win for a small invest.